recipe/patches/0016-Unvendor-openssl.patch

```diff
@@ -75,24 +75,24 @@ index b43c674..16d00fc 100644
 -    <Delete Files="@(_SSLDLL->'$(OutDir)%(Filename)%(Extension)')" TreatErrorsAsWarnings="true" />
 -  </Target>
 -</Project>
-\ No newline at end of file
+\ No newline at end of file
 +</Project>
 diff --git a/PCbuild/python.props b/PCbuild/python.props
 index 8c8392d..670feeb 100644
 --- a/PCbuild/python.props
 +++ b/PCbuild/python.props
 @@ -49,9 +49,9 @@
      <sqlite3Dir>$(ExternalsDir)sqlite-3.31.1.0\</sqlite3Dir>
-     <bz2Dir>$(ExternalsDir)bzip2-1.0.6\</bz2Dir>
+     <bz2Dir>$(ExternalsDir)bzip2-1.0.8\</bz2Dir>
      <lzmaDir>$(ExternalsDir)xz-5.2.2\</lzmaDir>
--    <opensslDir>$(ExternalsDir)openssl-1.1.1g\</opensslDir>
--    <opensslOutDir>$(ExternalsDir)openssl-bin-1.1.1g\$(ArchName)\</opensslOutDir>
+-    <opensslDir>$(ExternalsDir)openssl-1.1.1n\</opensslDir>
+-    <opensslOutDir>$(ExternalsDir)openssl-bin-1.1.1n\$(ArchName)\</opensslOutDir>
 -    <opensslIncludeDir>$(opensslOutDir)include</opensslIncludeDir>
 +    <opensslDir>$(OPENSSL_DIR)\</opensslDir>

```

Comment:
can you explain in a few words why this change is necessary now? it uses an environment variable which is new.

Reply:
I have not changed line 91 with the OPENSSL_DIR variable, this has been in the patch file for a long time. It just looks that way because it's a diff of a patch file which Github doesn't display that well.

Reply:
understood. perfect. Thank you for pointing that out.

It seems that python changed the version of openssl they built against.

Our patch also previously replaced it with an environment variable.